code/game/objects/items/weapons/clown_items.dm

```diff
@@ -105,28 +105,30 @@
 	throw_speed = 3
 	throw_range = 7
 	attack_verb = list("HONKED")
-	var/spam_flag = 0
+	var/spam_flag = FALSE
 	var/honksound = 'sound/items/bikehorn.ogg'
 	var/cooldowntime = 20
 
 /obj/item/weapon/bikehorn/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] solemnly points the horn at [user.p_their()] temple! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.visible_message("<span class='suicide'>[user] solemnly points \the [src] at [user.p_their()] temple! It looks like [user.p_theyre()] trying to commit suicide!</span>")

```

Comment:
Please use the pronoun helpers found in [this file](https://github.com/HippieStation/HippieStation13/blob/master/code/__HELPERS/pronouns.dm)